src/features/courses/course-controller.ts

```diff
@@ -729,7 +729,13 @@ class CourseController {
                 }
             });
 
+            // If only one question is deleted, rather than a whole topic.
             if (!_.isNil(existingQuestion)) {
+                // If this question was previously in an error state, decrement the topic's error cache.
+                if (!_.isNil(existingQuestion.errors)) {
+                    existingQuestion.getTopic().then(topic => topic.decrement('errors'));
```

Comment:
Is decrement defined? I don't see it anywhere

Reply:
I see that the build check succeeded, I guess this is a sequelize function?

Reply:
Regardless we should not have orphaned promises without catches
We should add
```TypeScript
.catch(err => logger.error(`Could not decrement topic errors: ${err.message}`));
```

Reply:
It's a sequelize function, yeah. Do we need to explicitly catch errors like those? I thought if they bubbled up, the transaction would be rolled back and we'd still get a Sequelize error on the logs.